backend/src/test/java/com/github/astridottmann/backend/controllers/MongoUserIntegrationTest.java

```diff
@@ -1,25 +1,33 @@
 package com.github.astridottmann.backend.controllers;
 
+import com.github.astridottmann.backend.models.MongoUser;
+import com.github.astridottmann.backend.repositories.MongoUserRepository;
 import org.junit.jupiter.api.Test;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
 import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.http.MediaType;
 import org.springframework.security.test.context.support.WithMockUser;
+import org.springframework.test.annotation.DirtiesContext;
 import org.springframework.test.web.servlet.MockMvc;
 
 import static org.springframework.security.test.web.servlet.request.SecurityMockMvcRequestPostProcessors.csrf;
 import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get;
 import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.post;
-import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.content;
-import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.*;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.jsonPath;
 
 @SpringBootTest
 @AutoConfigureMockMvc
+@DirtiesContext(classMode = DirtiesContext.ClassMode.BEFORE_EACH_TEST_METHOD)
 class MongoUserIntegrationTest {
 
     @Autowired
     MockMvc mockMvc;
 
+    @Autowired
+    MongoUserRepository mongoUserRepository;
```

Comment:
I think this is not a good approach. In the integration test we should use the endpoints to manipulate the data.
Perhaps I'm wrong, because you use it for a special use.

Reply:
I use it, to add an element to the repo for the sign-in-test. I was thinking, that is okay, because I need a user in the database.